modules/tinymce/src/themes/silver/main/ts/ui/menus/item/build/ColorSwatchItem.ts

```diff
@@ -10,7 +10,7 @@ import { Menu } from '@ephox/bridge';
 import { Id } from '@ephox/katamari';
 
 import { UiFactoryBackstage } from 'tinymce/themes/silver/backstage/Backstage';
-import ColorSwatch from 'tinymce/themes/silver/ui/core/color/ColorSwatch';
+import * as ColorSwatch from 'tinymce/themes/silver/ui/core/color/ColorSwatch';
```

Comment:
We really shouldn't be using an alias import here, but it was like this before so a task for another day :grinning:

Reply:
Logged TINY-4724 to look at this.